src/executors/large.yml

```diff
@@ -1,5 +1,5 @@
 description: >
   Executor for more demanding jobs.
 docker:
-  - image: 'figurepos/aws-node-terraform:node20'
+  - image: 'figurepos/aws-node-terraform:525cd89'
```

Comment:
To je ten aktualizovaný image: https://github.com/FigurePOS/circle-ci-build-image/commit/525cd89f7b067269ae8ba32441c2f0861c9a2fc0